packages/manager/Dockerfile

```
@@ -0,0 +1,27 @@
+# `manager`
+#
+# Serves Cloud Manager.
+FROM node:14.17-bullseye-slim as manager
+ENV NODE_ENV=development
+WORKDIR /home/node/app
+VOLUME /home/node/app
+EXPOSE 3000/tcp
+# Curl is needed for health check.
+RUN apt-get update && apt-get install -y curl
```

Cleaning up after installing will help reduce the image size. We do this in many of our other Dockerfiles. I would look around at other example projects to see what they're doing but here are some quick suggestions!


```suggestion
RUN apt-get update
    && apt-get install -y curl \ 
    && rm -rf /var/cache/apt/* \
    && rm -rf /var/lib/apt/lists/* \
    && apt-get clean
```